WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationActivity.kt

```diff
@@ -170,6 +177,12 @@ class SiteCreationActivity : LocaleAwareActivity(),
     }
 
     private fun showStep(target: WizardNavigationTarget<SiteCreationStep, SiteCreationState>) {
+        // Cancel preload job before displaying the theme picker.
+        if (target.wizardStep == SITE_DESIGNS) {
+            mainViewModel.preloadingJob?.cancel(
+                    "Preload did not complete before theme picker was shown."
```

Comment:
In my testing of the interrupted flow, I did not observe this in the logs.. it is possible that the job cancellation is not working as intended (though the UX was still ok, imo). Still, it may be worth exploring further why this did not appear in the following scenario: the site intent screen is skipped immediately, presenting the theme picker fragment before the mshots have completed preloading. Curious if you have any ideas why this doesn't get logged?

Reply:
I haven't been able to get the cancelation message either. The functionality seems to work thought 👍 
On a side note (this is mostly personal preference), we could move this block in the `when` clause below. Wdyt?

Reply:
Thanks for reporting that you had a similar experience :+1:  I moved the cancellation with https://github.com/wordpress-mobile/WordPress-Android/pull/16584/commits/ba5f8c1638520457b02a33c3b6180191af36b4c2